lib/core/src/lib/viewer/components/viewer.component.spec.ts

```diff
@@ -525,6 +525,34 @@ describe('ViewerComponent', () => {
             });
         });
 
+        describe('Close Button', () => {
+
+            it('should show close button on left side when showCloseButton is true and allowGoBack is true', () => {
```

Comment:
You can even see from those unit tests that `allowGoBack` now works differently which should not be the case

Reply:
Done